css/style.css

```diff
@@ -133,6 +164,28 @@ a[href*="/sociallogin/oauth/"]::before {
   background-image: url(../img/button/salesforce.svg);
   width: 24px;
 }
+
 #alternative-logins a.button.salesforce {
   background-color: #009EDB;
 }
+
+@media only screen and (max-width: 1024px) {
+  #alternative-logins a.button[href*="/sociallogin/oauth/facebook"]::before,
+  #alternative-logins a.button[href*="/sociallogin/oauth/google"]::before,
+  #alternative-logins a.button[href*="/sociallogin/oauth/amazon"]::before,
+  #alternative-logins a.button[href*="/sociallogin/oauth/twitter"]::before,
+  #alternative-logins a.button[href*="/sociallogin/oauth/GitHub"]::before,
+  #alternative-logins a.button[href*="/sociallogin/oauth/discord"]::before,
+  #alternative-logins a.button::before,
```

Comment:
This selector should be enough

Reply:
Thanks, reduced it to a single selector. :-)